extensions/azure/azure-test/src/testFixtures/java/org/eclipse/dataspaceconnector/azure/testfixtures/CosmosTestClient.java

```diff
@@ -27,7 +28,7 @@
 
     static CosmosClient createClient() {
         var cosmosKey = propOrEnv("COSMOS_KEY", null);
```

Comment:
from where COSMOS_KEY values comes now as above you removed it from the pipeline environment variable ?

Reply:
Added doc in markdown